ts/Dashboards/Components/Component.ts

```diff
@@ -1124,7 +1116,14 @@ namespace Component {
              *
              * @default false
              */
-            isActive?: boolean;
+            activateComponent?: boolean;
+
```

Comment:
Are we decided on this name? It sounds a bit odd because it's a verb (suggests a callback?) when all the options are usually adjectives.